README.md

```diff
@@ -64,6 +64,14 @@ In self-hosted mode, dapr can be initialized using the CLI  with the placement,
 
 ([Prerequisite](#Prerequisites): Docker is available in the environment - recommended)
 
+```shell
```

Comment:
This should probably go under its own special section, no? something like

```
### Use Private Helm Repository

export HELM_CHART_REPO_URL="https://helmchart-repo.xxx.xxx/dapr/dapr"
export HELM_CHART_REPO_USERNAME="username_xxx"
export HELM_CHART_REPO_PASSWORD="passwd_xxx"

Setting the above parameters will allow `dapr init -k` to install Dapr images from the configured Helm repository.
```

Reply:
ok, i change to this. Thanks.

yanon2, help me check this proposal [#dapr/issues/4065](https://github.com/dapr/dapr/issues/4065)